App/views.py

```diff
@@ -0,0 +1,66 @@
+import json
+import requests
+import urllib.parse
+
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.utils.http import urlquote
+
+
+def get_article_view_count(request):
+    if request.method=='POST':
+        username = request.POST['username']
+        try:
+            if not username:
+                raise Error("Please enter a username")
```

Comment:
This should be Exception, not Error (I probably remembered wrong). Or you can [define your own error type](https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions) (that has the benefit that you can let other types bubble up to the framework and it will probably show a more developer-friendly error message, with stack traces and whatnot).